tests/integration/helm_default_installation_test.go

```diff
@@ -27,10 +27,10 @@ import (
 	"github.com/SumoLogic/sumologic-kubernetes-collection/tests/integration/internal/stepfuncs"
 )
 
-func Test_Helm_Default_FluentD_Metadata(t *testing.T) {
+func Test_Helm_Default_Otel_Metadata(t *testing.T) {
```

Comment:
If you're going to change this, you should modify the existing otel metadata test to use fluentd instead as well.

Reply:
I changed this because the default metadata provider was changed to otel. ok I'll ensure we have tests for both fluentd and otc metadata installation.